TYPO3.Neos/Resources/Private/Partials/Module/Shared/NodeContentDimensionsInformation.html

```diff
@@ -4,9 +4,9 @@
 		<f:if condition="{dimension.name} == {contentDimensionKey}">
 			<f:for each="{contentDimension.presets}" key="presetKey" as="preset">
 				<f:if condition="{presetKey} == {dimension.value}">
-					<span class="neos-label" title="<i class='{contentDimension.icon}'></i> {neos:backend.translate(id: contentDimension.label)}: {presetKey}" data-neos-toggle="tooltip" data-html="1">{neos:backend.translate(id: preset.label)}</span>
+					<span class="neos-label" title="{neos:backend.translate(id: 'content.dimension.preset')}: {neos:backend.translate(id: contentDimension.label)}, {neos:backend.translate(id: 'value')}:  {presetKey}" data-neos-toggle="tooltip" data-html="1"><i class='{contentDimension.icon}'></i> {neos:backend.translate(id: preset.label)}</span>
```

Comment:
> not saying we can't change it, but it was carefully considered in the first place so would be good with some good arguments for changing it.. e.g. was it difficult to understand? because to me this introduces a lot of redundant information (putting implicit Preset, Value labels) all over

In the case of my client, some dimensions in different presets use the same label. So icon + label make sense, but label only is confusing. And making the label longer use more space than displaying the icon inline.

> didn't add the icon to the label in the first place to use less space and since it's not really necessary to understand the information..

I disagree on this argument, we don't know how many dimension a projet use, so we can not manage the space, check my comment bellow, the icon can help in some condition to make the label shorter and use less space.